### ISSUE TITLE
button_text weirdness

### ISSUE THREAD

**Issue by pjv** [maykar/custom-header#26]
Hi @maykar,

I just updated from CCH to CH and I'm re-working my config.

I have this:
```
button_text:
    options: '{{ hours12 }}:{{ minutesLZ }} {{ ampm }}'
```

And I’m somehow getting this in the header:
<img width="132" alt="2019-12-16 at 10 29 AM" src="https://user-images.githubusercontent.com/327716/70924399-0fbe5c80-1fef-11ea-9e88-98f2b23f5d18.png">

Even though the DOM looks like this:

<img width="604" alt="2019-12-16 at 10 29 AM" src="https://user-images.githubusercontent.com/327716/70924441-1cdb4b80-1fef-11ea-977a-3ece0ccd9863.png">

Edit: it seems to have to do with that `dir=“rtl”` attribute.

**Comment by maykar** [comment]
Should be fixed in latest release

**Comment by grizzlyjere** [comment]
I'm experiencing this issue under 1.20.  Is there a regression?

**Comment by maykar** [comment]
Yeah this worked it's way back in, but should be fixed in 1.2.1

**Comment by pjv** [comment]
this is me in 1.2.1:

<img width="171" alt="2020-01-13 at 11 35 AM" src="https://user-images.githubusercontent.com/327716/72278259-3de88980-35f9-11ea-8b63-3f92ef8d50c9.png">

**Comment by maykar** [comment]
Can you confirm this isn't a cache issue by checking the version number reported in your browsers dev tools?

**Comment by maykar** [comment]
I only ask as on 1.2.1 using your same config I get:

![image](https://user-images.githubusercontent.com/25127328/72303738-3ac2bd00-363c-11ea-8258-1ced7a0a8e51.png)

**Comment by maykar** [comment]
Also, I haven't reopened this one since someone has made a duplicate that is still open. #96

**Comment by maykar** [comment]
Please see this comment and the following 2:

https://community.home-assistant.io/t/custom-header/155399/341

Seems there have been some issues when updating in HACS and it actually wasn't updating. Double check that you are on 1.2.1 and reinstall in HACS either way.

**Comment by pjv** [comment]
@maykar it's weird because I never installed 1.2.0 at all. I updated (via HACS) from 1.1.7 directly to 1.2.1 and saw the issue as in my screenshot above. I then rolled back to 1.1.7. Now, to answer your question I updated again to 1.2.1 and it's fine. Maybe HACS originally updated me to 1.2.0 even though I specified 1.2.1 in the dropdown and it (HACS) said after the update that I was on 1.2.1?

Anyway, I'm now on 1.2.1 (confirmed via js console) and it looks good in the header.
